bitbucket/pipelineService.js

```diff
@@ -0,0 +1,35 @@
+const createService = require('../utils').createService;
+const moment = require('moment');
+
+const url = (host) => {
+    return host + ['repositories', process.env.repo, 'pipelines/'].join('/');
+};
+
+const map = ({ build_number, created_on, repository, creator, state, target }) => {
+    const getTitle = () => {
+        let emoji = '✅';
+        const commitMessage = target.commit.message;
+
+        if (state.result && state.result.name === 'FAILED') {
+            emoji = '❌'
+        }
```

Comment:
I would prefer to have a function that returns the emoji instead of a value mutation, more like 
```js
const getEmoji = state => state.result.name === FAILED ? '❌' : '✅'
```
And please move the `'FAILED'` string literal to a constant.